src/Admin/AdminHelper.php

```diff
@@ -191,11 +187,10 @@ public function appendFormFieldElement(AdminInterface $admin, object $subject, s
      *
      * @param string $elementId expects string in format used in form id field.
      *                          (uniqueIdentifier_model_sub_model or uniqueIdentifier_model_1_sub_model etc.)
-     * @param mixed  $model
      *
      * @throws \Exception
      */
-    public function getElementAccessPath(string $elementId, $model): string
+    public function getElementAccessPath(string $elementId, object $model): string
```

Comment:
I ported that from @jordisala1991  PR, so I'm not sure if it should be mixed or or there were some changes in the code. But, looking now at `isReadable` which takes `$model` as a first parameter, it expects it to be `@param object|array`

Reply:
Yeah, I know you got that from jordi PR. 

I just added a comment in order to check if it's right. (I was on phone).

The only usage is
```
$this->getElementAccessPath($elementId, $subject);
```
Where `$subject` is coming from `appendFormFieldElement`

But now I think that `getElementAccessPath` should be a private method since it's only used in AdminHelper.